ui/components/Modal.js

```diff
@@ -0,0 +1,40 @@
+import { Modal, Backdrop, Fade } from '@mui/core'
+import { styled } from "@mui/material/styles";
+import { useTheme } from "@mui/system";
+
+const CustomModalWrapper = styled(Modal)(({ theme }) => ({
+   display : "flex",
+   alignItems : "center",
+   justifyContent : "center", 
+}))
+
+/**
+ *
+ * @param {{
+ *  open?: boolean,
+ *  Content?: JSX.Element,
+ *  handleClose?: (event: {}, reason: "backdropClick" | "escapeKeyDown") => void,
+ *  container?: React.ReactInstance | (() => React.ReactInstance)
+ * }} props
+ * @returns 
+ */
+
+export const CustomModal = ({
```

Comment:
Need not export here since you are exporting it using default export at the end.

Reply:
```suggestion
const CustomModal = ({
```

Reply:
committed the suggestion @sudo-NithishKarthik